README.md

```diff
@@ -217,7 +230,9 @@ target_link_libraries(
 )
 ```
 
-### Dependencies
+### Requirements
+A c++20 compatible compiler and standard library with concepts support.
+
 We depend on Boost's type_index package to get compile time generated ids for each type to be able to sort types for units (so we can compare types generated from different expressions).
 
 In the extensions subfolder we have added skills for common third party libraries: `fmt`, `absl` and `gtest`. Using any of these will ofcourse also require you to install those dependencies.
```

Comment:
In the extensions subfolder we have added skills for common third party libraries: `fmt`, `absl` and `gtest`. Using any of these will ofcourse also require you to install those dependencies.
-->

In the extensions subfolder we have added skills for common third party libraries: `fmt`, `absl` and `gtest`. using these also requires the relevant third party libraries to be installed.